src/services/copyConfig/rba/rba.js

```diff
@@ -84,14 +88,39 @@ export default class Rba {
     return response
   }
 
-  async setRbaRulesAndSettings(destinationSite, destinationSiteConfiguration, payload) {
-    const response = await this.#rbaPolicy.set(destinationSite, payload, destinationSiteConfiguration.dataCenter)
+  // Merge originCommonRules into destinationCommonRules by id
+  mergeCommonRules(originCommonRules, destinationCommonRules) {
+    const destinationCommonRulesMap = new Map(destinationCommonRules.map((rule) => [rule.id, rule]))
+
+    originCommonRules.forEach((originRule) => {
+      if (destinationCommonRulesMap.has(originRule.id)) {
+        const destinationRuleIndex = destinationCommonRules.findIndex((r) => r.id === originRule.id)
```

Comment:
do we need this find? 
why not just call destinationCommonRulesMap.set(rule.id, rule) to override in case it already exists or not exists and then return something like Array.from(destinationCommonRulesMap.values())?